pyface/ui/__init__.py

```diff
@@ -0,0 +1,84 @@
+# (C) Copyright 2005-2023 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+from importlib import import_module
+from importlib.abc import MetaPathFinder, Loader
+from importlib.util import find_spec
+from importlib.machinery import ModuleSpec
+import sys
+
+
+# Import hooks for loading pyface.ui.qt.* in place of a pyface.ui.qt4.*
+# This is just the implementation, it is not connected in this module, but
+# is available for applications which want to install it themselves.
+# It is here rather than in pyface.ui.qt4 so it can be imported and used
+# without generating the warnings from pyface.ui.qt4
+#
+# To use manually:
+#
+#     import sys
+#     sys.meta_path.append(ShadowedModuleFinder())
+
+class ShadowedModuleLoader(Loader):
+    """This loads another module into sys.modules with the given name.
+    """
+
+    def __init__(self, fullname, new_name, new_spec):
+        self.fullname = fullname
```

Comment:
Could we add a docstring (either here, or at class level)? `fullname` and `new_name` are somewhat clear, but it's not clear to me without consulting external sources what the type or contents of `new_spec` are expected to be.

Reply:
Done - I added to the class docstring.